lms/djangoapps/learner_home/serializers.py

```diff
@@ -48,7 +48,7 @@ class CourseProviderSerializer(serializers.Serializer):
 class CourseSerializer(serializers.Serializer):
     """Course header information, derived from a CourseOverview"""
 
-    bannerImgSrc = serializers.URLField(source="banner_image_url")
+    bannerImgSrc = serializers.URLField(source="image_urls.small")
```

Comment:
this source string is really generic.  We're confident this is the correct source?

Reply:
In the sense of, "this is what the current dashboard uses", yes.

Reply:
👍